### PR TITLE
Temporary changes that match project's logic.

### PR DESCRIPTION
**Pull request by DarkoNikolovski**
[21399](https://hive.versada.eu/projects/it-first-vsf/work_packages/21399/activity)

### CODE REVIEW

**Comment by DarkoNikolovski** [comment]
@oerp-odoo As we discussed this is only a temporary change until we refactor the module and implement the logic from the project into it.

File: graphql_vuestorefront/models/product.py [hunk 1]
```diff
@@ -7,11 +7,23 @@
...
     _inherit = 'product.template'
 
+    def _auto_init(self):
+        """Skip computation of the field on module installation
+        because it will fail as a result of large amount of products"""
+        if not column_exists(
+            self.env.cr, "product_template", "website_slug"
+        ):
+            create_column(
+                self.env.cr, "product_template", "website_slug", "varchar"
```
Comment (oerp-odoo) [inline graphql_vuestorefront/models/product.py:23]: You can add `SQL` here, to set same value as compute. This should not fail as its much more efficient than using compute.

File: graphql_vuestorefront/models/product.py [hunk 2]
```diff
@@ -44,18 +56,24 @@ def _compute_public_categ_slug_ids(self):
 
     @api.depends('name')
     def _compute_website_slug(self):
-        langs = self.env['res.lang'].search([])
-
+        """
+            Simplifying the method and removing lang from it.
+            IT First has the same product name for all languages
```
Comment (oerp-odoo) [inline graphql_vuestorefront/models/product.py]: I think better not use specific company names in a fork thats kind of still generic:)

File: graphql_vuestorefront/models/product.py (hunk 1 above)
Comment (DarkoNikolovski) [reply graphql_vuestorefront/models/product.py:23]: I would prefer if possible to not spend more time on this at the moment, will just run the compute method as a script to recalculate the values once is installed on the server.

**Review by oerp-odoo — APPROVED** [review]

**Pull request merged**